openapi/openapi.yaml

```diff
@@ -3087,6 +3090,18 @@ components:
               datetime:
                 type: string
                 example: 2021-02-23T09:29:45.254525
+              quay_repo_url:
+                type: string
+                nullable: true
+                example: https://quay.io/thoth-station/s2i-custom-notebook:v0.2.0
+              image_analysis_url:
+                type: string
+                nullable: true
+                example: https://khemenu.thoth-station.ninja/api/v1/analyze/package-extract-220106085109-984feaa8a3862285
+              thoth_search_url:
+                type: string
+                nullable: true
+                example: https://thoth-station.ninja/search/image/package-extract-220106085109-984feaa8a3862285
```

Comment:
I'm not sure if this should be exposed on the backend API endpoints. Can this URL be constructed on frontend?

Reply:
yes, it already is being constructed in the front end. All that is needed is the analysis id. Should this link not be included in the response?

Reply:
From a design point of view, it might be a good idea to keep user-api unaware of Thoth Search. That way, Thoth Search will be a consumer of endpoints.